core/tests/unit/ldml/keyboards/k_003_transform.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+from https://github.com/unicode-org/cldr/blob/keyboard-preview/docs/ldml/tr35-keyboards.md#element-transform
+
+Note: keys should start with [K_A][K_B][K_C] not [K_Y] when transforms are functioning properly.
+
+@@keys: [K_Y][K_BKQUOTE][K_A][K_B][K_D][K_BKQUOTE][K_A][K_B][K_E][K_Q][K_BKQUOTE][K_B][K_C]
+@@expected: y\u0020xd\u0020xeq\u0020m
+
+-->
+<!DOCTYPE keyboard SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/ldmlKeyboard.dtd">
+<keyboard locale="en" conformsTo="techpreview">
+  <info author="srl295" indicator="🙀" layout="qwerty" normalization="NFC" />
+
+  <names>
+    <name value="TestKbd" />
+  </names>
+
+  <keys>
+    <!-- <key id="gap" gap="true" /> -->
+    <key id="space" to=" " width="5" />
+
+    <!-- bug: these keys should not need to be specified -->
+    <key id="a" to="a" />
+    <key id="b" to="b" />
+    <key id="c" to="c" />
+    <key id="d" to="d" />
+    <key id="e" to="e" />
+    <key id="f" to="f" />
+    <key id="g" to="g" />
+    <key id="h" to="h" />
+    <key id="i" to="i" />
+    <key id="j" to="j" />
+    <key id="k" to="k" />
+    <key id="l" to="l" />
+    <key id="m" to="m" />
+    <key id="n" to="n" />
+    <key id="o" to="o" />
+    <key id="p" to="p" />
+    <key id="q" to="q" />
+    <key id="r" to="r" />
+    <key id="s" to="s" />
+    <key id="t" to="t" />
+    <key id="u" to="u" />
+    <key id="v" to="v" />
+    <key id="w" to="w" />
+    <key id="x" to="x" />
+    <key id="y" to="y" />
+    <key id="z" to="z" />
+    <key id="bquote" to="`"/>
+    <key id="backslash" to="\u{005c}"/>
+  </keys>
+
+  <layerMaps form="hardware">
+    <layerMap id="base">
+      <row keys="space" /> <!-- number row -->
+      <row keys="q w e r t y u i o p" />
+      <row keys="a s d f g h j k l" />
+      <row keys="backslash z x c v b n m" />
+      <!-- <row keys="space" /> --> <!-- Bug: 5 rows is 'too many'? -->
+    </layerMap>
+  </layerMaps>
+
+  <transforms type="simple">
+    <transform from="ab" to="x" />
+    <transform from="abc" to="y" />
```

Comment:
So when typing, this transform will never match because after typing `a` and `b` it's already replaced with `x`?

Reply:
right. 
Lots of issues w/ the spec and semantics of this one.